Lib/distutils/ccompiler.py

```diff
@@ -392,7 +392,7 @@ def _fix_compile_args(self, output_dir, macros, include_dirs):
         return output_dir, macros, include_dirs
 
     def _prep_compile(self, sources, output_dir, depends=None):
-        """Decide which souce files must be recompiled.
+        """Decide which source files must be recompiled.
```

Comment:
We generally dont touch the Python code since that's usually copied directly over from the CPython codebase.

Reply:
You can send the same fix to cpython project. Then we will bring it from their next release.

https://github.com/python/cpython

Reply:
Thank you! Let me fix them there too!